### DISCUSSION TITLE
Reférencement dans les stores

### DISCUSSION THREAD

**Discussion by TheTiger72** [PapillonApp/papillon-v5#97] [Ideas]
Bonjour, je ne sais pas si cela est possible, mais pour une meilleur visibilité de Papillon il faudrait être visible quand ont cherche "Pronote" dans les stores. Par exemple sur l'App Store il y'a l'ancienne application My Notes qui apparaît quand ont cherche le terme "Pronote".
Un meilleur référencement permettrait à plus de personnes de découvrir Papillon et de l'installer. Ce qui pourra favoriser les rapports de bug ou même les dons pour maintenir l'application.

**Comment by LeGeek01** [comment]
Approved by LeGeek

**Comment by moeenio** [comment]
Si t'as le secret du référencement on le veut bien \:)

**Comment by TheTiger72** [comment]
C'est pas à moi de trouver comment faire. Je propose l'idée c'est tout
Demander aux créateurs de My Notes, ils ont réussi à être référencé avec le mot "Pronote".

**Reply by moeenio** [reply]
Merci de proposer mais je ne crois pas que ce soit une idée à laquelle on n'ait jamais pensé. En fait ça me semble une évidence qu'il serait mieux que l'appli soit bien référencé.

**Reply by ecnivtwelve** [reply]
C'est rigolo que vous en parliez maintenant vu que hier même j'ai fait le même constat et j'ai modifié le sous-titre de l'App Store pour inclure le nom "Pronote".

![image](https://user-images.githubusercontent.com/32978709/230931418-dd95b27c-92bb-4a64-9697-2feeec3d4ab0.png)

Faut voir si ça va bien marcher !

**Reply by TheTiger72** [reply]
![image](https://user-images.githubusercontent.com/84013544/237006960-94afe3fe-8ba2-413e-8ef5-9f5160fa888d.jpeg)

**Reply by TheTiger72** [reply]
En descendant un peu l'appli est dispo avec le terme de recherche "Pronote" !

**Comment by Rexxt** [comment]
quelle est l'utilité finale ?

**Reply by TheTiger72** [reply]
Que plus de personnes installes Papillon, puisse reporter des bugs...etc

**Reply by moeenio** [reply]
la grosse moula

**Comment by Ninjdai1** [comment]
Bonjour !
Pour le moment, j'installe Papillon directement depuis l'apk des releases.
Verra-t-on un jour une version sur [F-droid](https://fdroid.org) ?

**Reply by TheTiger72** [reply]
Utilise le PlayStore sinon...

**Reply by moeenio** [reply]
J'avais discuté de ça en privé avec @ecnivtwelve et il n'a pas rejeté l'idée, mais il faut savoir que les soumissions sur F-Droid demandent un peu d'effort car il faut s'assurer que leur serveur puisse compiler l'application à partir du dépôt de code source, notamment dans le cas des applications web hybrides comme ici où ils ne gèrent pas la compilation/bundling du JavaScript etc voir [FAQ sur les buildsystems avec note à propos de Cordova](https://f-droid.org/fr/docs/FAQ_-_App_Developers/#which-build-system-to-use) qui s'applique certainement pour Capacitor).

On pourrait éventuellement commencer par être présent sur [le dépôt de IzzyOnDroid](https://apt.izzysoft.de/fdroid/index/info) qui récupère les apk compilés par les dévelopeurs d'applications libres.

Sinon tu peux utiliser [Obtainium](https://github.com/ImranR98/Obtainium) pour suivre les mises à jour à partir de GitHub.

**Reply by Ninjdai1** [reply]
> Utilise le PlayStore sinon...

J'utilise F-Droid pour une raison 😅

**Reply by Ninjdai1** [reply]
> J'avais discuté de ça en privé avec @ecnivtwelve et il n'a pas rejeté l'idée, mais il faut savoir que les soumissions sur F-Droid demandent un peu d'effort car il faut s'assurer que leur serveur puisse compiler l'application à partir du dépôt de code source, notamment dans le cas des applications web hybrides comme ici où ils ne gèrent pas la compilation/bundling du JavaScript etc voir [FAQ sur les buildsystems avec note à propos de Cordova](https://f-droid.org/fr/docs/FAQ_-_App_Developers/#which-build-system-to-use) qui s'applique certainement pour Capacitor).
> 
> On pourrait éventuellement commencer par être présent sur [le dépôt de IzzyOnDroid](https://apt.izzysoft.de/fdroid/index/info) qui récupère les apk compilés par les dévelopeurs d'applications libres.
> 
> Sinon tu peux utiliser [Obtainium](https://github.com/ImranR98/Obtainium) pour suivre les mises à jour à partir de GitHub.

Je comprends, merci de ta réponse, j'y jetterai un oeil !

**Reply by TheTiger72** [reply]
> > Utilise le PlayStore sinon...
> 
> J'utilise F-Droid pour une raison 😅

Laquel ? Sauf si c'est indiscret

**Reply by moeenio** [reply]
Ça peut être un besoin de confidentialité, être anticapitaliste, avoir un appareil chelou sans google play...

**Reply by Ninjdai1** [reply]
> Ça peut être un besoin de confidentialité, être anticapitaliste, avoir un appareil chelou sans google play...

Les deux premières, et j'apprécierai la troisième X)
